src/sentry/relay/config/__init__.py

```diff
@@ -246,7 +247,7 @@ class CardinalityLimit(TypedDict):
     namespace: str | None
 
 
-def get_metrics_config(project: Project) -> Mapping[str, Any] | None:
+def get_metrics_config(timeout: BuildTimeChecker, project: Project) -> Mapping[str, Any] | None:
```

Comment:
Does the linter complain when somebody ignores this parameter?

Reply:
Yep, I expanded the PR description with the error message. Let me know if I can clarify it further